.wsignore

```diff
@@ -0,0 +1,9 @@
+# WayScript recommended defaults
+node_modules
+**/node_modules
+.DS_STORE
+**/.DS_Store
+__pycache__
+.git
+**/.git
```

Comment:
If this works like .gitignore then `node_modules` includes all files/folders by that name (including `**/node_modules`)
```suggestion
node_modules
.DS_STORE
__pycache__
.git
```
@jeffbuswell I couldn't find the paring logic for this file.  Do you know if that's correct?

Reply:
I didn't make this file, it came automatically from creating the lair